actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -470,22 +471,21 @@ private void courseConsumptionMetrics(Request actorMessage) {
       }
 
       String channel = (String) rootOrgData.get(JsonKey.HASHTAGID);
-      ProjectLogger.log("Channel" + channel);
-      String responseFormat =
-          (String) cache.getData(JsonKey.CourseConsumption, courseId, periodStr);
-      if (StringUtils.isBlank(responseFormat)) {
-        responseFormat = getCourseConsumptionData(periodStr, courseId, requestObject, channel);
-        cache.setData(JsonKey.CourseConsumption, courseId, periodStr, responseFormat);
-      }
+      ProjectLogger.log(
+          "CourseMetricsActor:courseConsumptionMetrics organisation hashtag id." + channel,
+          LoggerEnum.INFO.name());
+      String responseFormat = getCourseConsumptionData(periodStr, courseId, requestObject, channel);
       Response response =
           metricsResponseGenerator(responseFormat, periodStr, getViewData(courseId));
       sender().tell(response, self());
     } catch (ProjectCommonException e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "CourseMetricsActor:courseConsumptionMetrics exception occurs " + e.getMessage(), e);
       sender().tell(e, self());
       return;
     } catch (Exception e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "CourseMetricsActor:courseConsumptionMetrics exception occurs " + e.getMessage(), e);
```

Comment:
From:
"CourseMetricsActor:courseConsumptionMetrics exception occurs "

To:
"CourseMetricsActor:courseConsumptionMetrics: Generic exception in getting course consumption data: "